src/main/resources/solr/anserini/conf/stopwords_en.txt

```diff
@@ -12,3 +12,43 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# a couple of test stopwords to test that the words are really being
```

Comment:
should we remove these couple of lines?

add a comment at the beginning saying this is Lucene's default?

Reply:
sure I'll remove these lines.
It thought the line after is already stating that this is Lucene's default? -
https://github.com/castorini/anserini/pull/1345/files/5fff9e78959a7a578f12d8d816c1f7052ca4228c#diff-22892be18d725ce8edba42553448193fR21